Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceRemediation/MSFT_IntuneDeviceRemediation.schema.mof

```diff
@@ -35,8 +54,8 @@ class MSFT_IntuneDeviceRemediation : OMI_BaseResource
     [Write, Description("Indicate whether PowerShell script(s) should run as 32-bit")] Boolean RunAs32Bit;
     [Write, Description("Indicates the type of execution context. Possible values are: system, user."), ValueMap{"system","user"}, Values{"system","user"}] String RunAsAccount;
     [Key, Description("The unique identifier for an entity. Read-only.")] String Id;
-    [Write, Description("Represents the assignment to the Intune policy."), EmbeddedInstance("MSFT_DeviceManagementConfigurationPolicyAssignments")] String Assignments[];
-    [Write, Description("Present ensures the policy exists, absent ensures it is removed.")] string Ensure;
+    [Write, Description("Represents the assignment to the Intune policy."), EmbeddedInstance("MSFT_IntuneDeviceRemediationPolicyAssignments")] String Assignments[];
```

Comment:
I believe this would be a breaking change in the resource. Existing configurations that include Assignments[] as DeviceManagementConfigurationPolicyAssignments would be broken by this change.

Reply:
I understand, but that's unfortunately a necessity for this change. But the resource never actually worked with the assignments previously, so the configuration could be exported (with wrong assignments), and never correctly deployed. In my opinion, since the resource is relatively new and because of the fact that the assignments were broken, there should be the possibility to allow this change to be deployed.

Reply:
@NikCharlebois Yes, Fabien created this resource just 1 month ago and had this problem I reported with the display name not being present, or let's call it didn't have this feature, but actually was also missing the run schedule so the deployed scripts wouldn't be of much use anyway.

Reply:
I agree with your observation. Since the resource never actually work, we don't risk breaking it further. I will review and merge in today's release. Thanks for all your help Fabien, really appreciated.

Reply:
Thanks a bunch for the understanding, glad I can help.